plugins/woocommerce/client/admin/client/settings-payments/settings-payments-methods.scss

```diff
@@ -1 +1,111 @@
 @import "settings-payments-main.scss";
+
+.settings-payments-methods__container {
+	// Temporarily copy styles while things are under construction.
+	@extend .settings-payments-main__container;
+
+	.settings-payments-methods__show-more {
+		display: block;
+		cursor: pointer;
+		margin-top: $gap;
+		margin-left: 38px;
+		box-shadow: none;
+		color: var(--wp-admin-theme-color);
+	}
+
+	// Override styles for woocommerce list items.
+	.woocommerce-list {
+		border-top: 1px solid $gray-200;
+		border-bottom: 1px solid $gray-200;
+
+		&__item {
+			padding: 0 48px;
+
+			&:hover {
+				background-color: $white;
+
+				.woocommerce-list__item-title {
+					color: $gray-900;
+				}
+			}
+
+			&:not(:first-child) {
+				border-top: 1px solid $gray-200;
+			}
+		}
+
+		&__item.clickable-list-item:hover {
+			background-color: rgba(0, 0, 0, 0.02); /* Light overlay effect */
+		}
+
+		&__item-inner {
+			padding: $gap-small 0;
+			align-items: stretch;
+		}
+
+		&__item-inner:focus {
+			box-shadow: none;
+		}
+
+		&__item-title {
+			font-size: 16px;
+			line-height: 24px;
+			color: $gray-900;
+			font-weight: 600;
+			display: inline-flex;
+			gap: 10px;
+			align-items: center;
+		}
+
+		&__item-content {
+			font-size: 13px;
+			font-weight: 400;
+			line-height: 20px;
+			color: $gray-700;
+
+			.woocommerce-woopayments-payment-methods-logos {
+				flex-wrap: wrap;
+			}
+		}
+
+		&__item-after {
+			&__actions {
+				display: flex;
+				gap: $gap;
+
+				.components-base-control {
+					margin-bottom: 0;
+				}
+
+				.components-button {
+					height: 32px;
+				}
+			}
+		}
+
+		&__item-before {
+			img {
+				max-width: 96px;
+				height: 40px;
+				border-radius: 4px;
+			}
+		}
+
+		&__item-multi-row {
+			display: flex;
+
+			&.multi-row-space {
+				margin-bottom: $gap-large;
+			}
+		}
+	}
+}
+
+#wpbody:has(.settings-payments-methods__container) {
+	margin-top: 0;
+}
+
+/* Use this carefully as in changes the body background to white. Include it only in components, otherwise white background will be all over the pages. */
+#wpcontent {
+	background-color: #fff;
```

Comment:
@mordeth we have `settings-payments-body.scss` for that, you could just include it

Reply:
Thank you! Apologies for missing that file earlier. It’s fixed now.

Reply:
no worries! thanks for addressing 🚀